src/components/Trending.jsx

```diff
@@ -1,78 +1,106 @@
-import axios from 'axios'
+import React, { useEffect, useState } from "react";
+import {
+  Radar,
+  RadarChart,
+  PolarGrid,
+  PolarAngleAxis,
+  PolarRadiusAxis,
+  AreaChart,
+  Area,
+  XAxis,
+  YAxis,
+  CartesianGrid,
+  Tooltip,
+  ResponsiveContainer,
+} from "recharts";
+import axios from "axios";
 
 function Trending() {
-    const [data,setData]= useState([]);
-   
-    useEffect(()=>{
-        axios.get("https://api.coingecko.com/api/v3/search/trending")
-        .then(response=>{
-            
-            setData([...response.data.coins])
-            // console.log(...response.data.coins)
-        })
-    },[])
+  const [data, setData] = useState([]);
+
+  useEffect(() => {
+    axios
+      .get("https://api.coingecko.com/api/v3/search/trending")
+      .then((response) => {
+        setData([...response.data.coins]);
+        // console.log(...response.data.coins)
+      });
+  }, []);
   return (
     <div>
-        <div style={{display:"flex",justifyContent:"center",alignItems:"center",width:'100vw'}}>
-        <AreaChart
-    width={800}
-    height={300}
-    data={data}
-    margin={{
-      top: 10,
-      right: 30,
-      left: 0,
-      bottom: 0
-    }}
-  >
-    <CartesianGrid strokeDasharray="3 3" />
-    <XAxis dataKey="item.name" />
-    <YAxis dataKey="item.market_cap_rank" reversed="true"/>
-    <Tooltip />
-    <Area type="monotone" dataKey="item.market_cap_rank" stroke="#8884d8" fill="#8884d8" />
-
-  </AreaChart>
-  <RadarChart
-      cx={300}
-      cy={250}
-      outerRadius={150}
-      width={500}
-      height={500}
-      data={data}
-    >
-      <PolarGrid />
-      <PolarAngleAxis dataKey="item.name" />
-      <PolarRadiusAxis />
-      <Radar
        
-        dataKey="item.price_btc"
-        stroke="#8884d8"
-        fill="#8884d8"
-        fillOpacity={0.6}
-      />
-    </RadarChart>
-        </div>
+        
+        <div style={{ display: "flex",height:"20vh",flexWrap:"wrap",width:"70%",alignItems:"center",justifyContent:"center"}}>
+        {data.map((item) => {
+          return (
             
-    <div>
-        {data.map((item)=>{
-            return(
+              <div style={{ display: "flex" }}>
+                <img src={item.item.small} alt="" />
                 <div>
-                    <p>{item.item.name}({item.item.symbol})</p>
-                    <p>{item.item.price_btc} BTC</p>
-                    <img src={item.item.small} alt="" />
+                  <p>
+                    {item.item.name}({item.item.symbol})
+                  </p>
+                  <p>{item.item.price_btc.toFixed(15)} BTC</p>
                 </div>
-            )
+            
+              </div>
+            
+          );
         })}
+      </div>
+        
+     
+      <div
+        style={{
+          display: "flex",
+          justifyContent: "center",
+          alignItems: "center",
+          width: "100vw",
+        }}
+      >
+        <AreaChart
```

Comment:
Create this as a separate component under charts folder and pass data as props

Reply:
okay sir by tmwr